### ISSUE TITLE
Python UDFs - class or not?

### ISSUE THREAD

**Issue by ruebot** [archivesunleashed/aut#467]
I implemented the UDFs on the Python side as a class. Do we want to leave them that way in advance of the 1.0.0 release, or do we want them stand-alone?

Basically, do we want this:

```python
  from aut import *
  
  WebArchive(sc, sqlContext, "/path/to/warcs")\
    .webpages()\
    .select("crawl_date", Udf.extract_domain("url").alias("domain"), "url", Udf.remove_html("content").alias("content"))
    .write.csv("plain-text-df/")
```

Or this:

```python
  from aut import *
  
  WebArchive(sc, sqlContext, "/path/to/warcs")\
    .webpages()\
    .select("crawl_date", extract_domain("url").alias("domain"), "url", remove_html("content").alias("content"))
    .write.csv("plain-text-df/")
```

**Comment by ianmilligan1** [comment]
I'm relatively agnostic, but from a user perspective it would be nice to have them as stand-alone (i.e. the second example above).

**Comment by ruebot** [comment]
Agreed. Then, the examples would all be pretty much the same between Scala and Python for most of the UDFs.

**Comment by ruebot** [comment]
:man_facepalming: we can just use the `from` functionally as @lintool mentioned in the call today. 

I'll get the documentation updated, and once that is merged, we can close this issue.

**Comment by ruebot** [comment]
Actually, we're already doing the `from` import. @lintool did you mean something else that I'm missing?

**Comment by lintool** [comment]
I prefer just `extract_domain` over `Udf.extract_domain`.
